app/views/managers/index.html.erb

```diff
@@ -1 +1,59 @@
-<h1>Manager Portal</h1>
\ No newline at end of file
+<h1>Manager Portal</h1>
+
+
+<div class="d-flex justify-content-center mt-5">
+  <%# debugger %>
+  <h1>Welcome to Reimbursement Management System </h1>
+</div>
+
+<div class="d-flex justify-content-center mt-5">
+  <h1><%= current_user.roles.first._type.titleize %></h1>
+</div>
+
+<div class="d-flex justify-content-center mt-5" >
+  <div class="card text-center mx-5" style="width:400px">
+    <%#= image_tag @user.image if @user.image.attached? %>
+    <h4 class="text-center">Previously Submitted Forms</h4>
+    <div class="card-body">
+      <h4 class="card-title"><%= current_user.name %></h4>
+      <p class="card-text">Some example text some example text. John Doe is an architect and engineer
+        Some example text some example text. John Doe is an architect and engineer</p>
+
+      <div class = "btn btn-primary mr-3">
+        <%= link_to "Check Out", destroy_user_session_path, :class => "btn" %>
+      </div>
+    </div>
+  </div>
+
+
+  <div class="card text-center mx-5" style="width:400px">
+    <%#= image_tag @user.image if @user.image.attached? %>
+    <h4 class = "text-center">Apply For New Form</h4>
+    <div class="card-body">
```

Comment:
remove all commented code and also try using partials for the view that is repeating